### PR TITLE
New package: EmptyFlow.ArdorQuery version 0.0.12

### PR DESCRIPTION
**Pull request by trueromanus**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132345)

### CODE REVIEW

**Comment by trueromanus** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\validator\AppData\Roaming\Microsoft\Installer\{EBE34EE1-85DB-4115-A8EB-59FD029384B7}\_853F67D554F05449430E7E.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Users\validator\AppData\Roaming\Microsoft\Installer\{EBE34EE1-85DB-4115-A8EB-59FD029384B7}\_853F67D554F05449430E7E.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions) Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\validator\AppData\Roaming\Microsoft\Installer\{EBE34EE1-85DB-4115-A8EB-59FD029384B7}\_957439F6BE687D2FF9FE82.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Users\validator\AppData\Roaming\Microsoft\Installer\{EBE34EE1-85DB-4115-A8EB-59FD029384B7}\_957439F6BE687D2FF9FE82.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform. Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\validator\AppData\Roaming\Microsoft\Installer\{EBE34EE1-85DB-4115-A8EB-59FD029384B7}\_6AA6732422F331E7E22BEA.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Users\validator\AppData\Roaming\Microsoft\Installer\{EBE34EE1-85DB-4115-A8EB-59FD029384B7}\_6AA6732422F331E7E22BEA.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.

(Automated response - build 515.)

**Comment by stephengillie** [comment]
image

File: manifests/e/EmptyFlow/ArdorQuery/0.0.12/EmptyFlow.ArdorQuery.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: EmptyFlow.ArdorQuery
+PackageVersion: 0.0.12
+InstallerLocale: en-US
+InstallerType: msi
+ProductCode: '{EBE34EE1-85DB-4115-A8EB-59FD029384B7}'
```
Comment (stephengillie) [inline manifests/e/EmptyFlow/ArdorQuery/0.0.12/EmptyFlow.ArdorQuery.installer.yaml:8]:
```suggestion
ProductCode: '{EBE34EE1-85DB-4115-A8EB-59FD029384B7}'
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by stephengillie** [comment]
Hi @trueromanus,

The package installs correctly, but has a dependency on `Microsoft.VCRedist.2015+.x64` to launch.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**